modules.json

```diff
@@ -367,4 +367,16 @@
             "ucuser": "Jimbo Wales"
     	}
     }
+    {
+        "filename": "get_allrevisions.py",
+        "docstring": "Demo of `AllRevisions` module: List All Revisions of all pages and all users.",
+        "endpoint": "https://en.wikipedia.org/w/api.php",
+        "params":{
+    		"action": "query",
+    		"format": "json",
+    		"list": "allrevisions",
```

Comment:
This example should show how to make a get request to fetch a list of all revisions by user `Place holder` on the English Wikipedia. Add any necessary parameters that would be required for this.